client/lxd_cluster.go

```diff
@@ -113,6 +113,11 @@ func (r *ProtocolLXD) UpdateClusterMember(name string, member api.ClusterMemberP
 	if !r.HasExtension("clustering_edit_roles") {
 		return fmt.Errorf("The server is missing the required \"clustering_edit_roles\" API extension")
 	}
+	if member.FailureDomain != "" {
+		if !r.HasExtension("clustering_failure_domains") {
+			return fmt.Errorf("The server is missing the required \"clustering_failure_domains\" API extension")
+		}
+	}
```

Comment:
Please make that its own commit. Changes to `client/` are always backported.

Reply:
Done.